src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -245,6 +246,17 @@ public function startTest($test)
 
     public function endTest($test, $time)
     {
+        if ($file = getenv('SYMFONY_EXPECTED_DEPRECATIONS_SERIALIZE')) {
+            putenv('SYMFONY_EXPECTED_DEPRECATIONS_SERIALIZE');
+            $expected_deprecations = file_get_contents($file);
```

Comment:
please use camelCase for variable names.

Reply:
Oops - Drupal habits die hard. Thanks for the review :)

Reply:
you forgot part of them